scripts/test/Selenium/Agent/Admin/AdminSort.t

```diff
@@ -50,16 +50,20 @@ $Selenium->RunTest(
             @NavigationCheck = (
                 'Általános ügyintéző',
                 'Dinamikus mezők',
+                'Dynamic fields ↔ Screens',
```

Comment:
I assume you overlooked this one?
```System file support```

Reply:
@tipue-dev damn, you're right.
We can use only 

```
my @NavigationCheck = (
    'Csomagkezelő',
    'Munkamenet-kezelés',
    'Naptárak',
    'Rendszerbeállítások',
    'Rendszerkarbantartás',
    'Rendszernapló',
    'SQL doboz',
  );

```

This should also work. 
Otherwise we'll just test with german translations.
The question is, why specifically Hungarian is tested here.